c_api/include/taichi/cpp/taichi.hpp

```diff
@@ -663,60 +685,6 @@ class AotModule {
   }
 };
 
-class Event {
```

Comment:
Are we intended to deprecate the C++ wrapper for `TiEvent` or is it by mistake?

Reply:
yes, it was introduced as a hack for unity integration but we no longer need it now. but anyway, let's defer it to another or